File: swagger/models/service.py
from .base_model import BaseModel


class Service(BaseModel):
	"""
	no description available

	Attributes:

		price
		The cost of the pricing option when sold at a physical location.

		online_price
		The cost of the pricing option when sold online.

		tax_included
		The amount of tax included in the price, if inclusive pricing is
		enabled.

		program_id
		The ID of the program that this pricing option applies to.

		tax_rate
		The tax rate applied to the pricing option. This field is populated
		only when you include a `LocationID` in the request.

		product_id
		The unique ID of the pricing option.

		id
		The barcode ID of the pricing option.

		name
		The name of the pricing option.

		count
		The initial count of usages available for the pricing option.
	"""

	input_map = {
		'Price': 'price',
		'OnlinePrice': 'online_price',
		'TaxIncluded': 'tax_included',
		'ProgramId': 'program_id',
		'TaxRate': 'tax_rate',
		'ProductId': 'product_id',
		'Id': 'id',
		'Name': 'name',
		'Count': 'count',
		}

	output_map = {
		'price': 'Price',
		'online_price': 'OnlinePrice',
		'tax_included': 'TaxIncluded',
		'program_id': 'ProgramId',
		'tax_rate': 'TaxRate',
		'product_id': 'ProductId',
		'id': 'Id',
		'name': 'Name',
		'count': 'Count',
		}

File: swagger/Models/Service.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class Service extends BaseModel
{
	/**
	 * The cost of the pricing option when sold at a physical location.
	 * 
	 * @var number format:double
	 */
	public $price;
	/**
	 * The cost of the pricing option when sold online.
	 * 
	 * @var number format:double
	 */
	public $onlinePrice;
	/**
	 * The amount of tax included in the price, if inclusive pricing is
	 * enabled.
	 * 
	 * @var number format:double
	 */
	public $taxIncluded;
	/**
	 * The ID of the program that this pricing option applies to.
	 * 
	 * @var integer format:int32
	 */
	public $programId;
	/**
	 * The tax rate applied to the pricing option. This field is populated
	 * only when you include a `LocationID` in the request.
	 * 
	 * @var number format:double
	 */
	public $taxRate;
	/**
	 * The unique ID of the pricing option.
	 * 
	 * @var integer format:int32
	 */
	public $productId;
	/**
	 * The barcode ID of the pricing option.
	 * 
	 * @var string 
	 */
	public $id;
	/**
	 * The name of the pricing option.
	 * 
	 * @var string 
	 */
	public $name;
	/**
	 * The initial count of usages available for the pricing option.
	 * 
	 * @var integer format:int32
	 */
	public $count;

	protected function getInputMap()
	{
		return [
			'Price' => ['price', 'number', 'double'],
			'OnlinePrice' => ['onlinePrice', 'number', 'double'],
			'TaxIncluded' => ['taxIncluded', 'number', 'double'],
			'ProgramId' => ['programId', 'integer', 'int32'],
			'TaxRate' => ['taxRate', 'number', 'double'],
			'ProductId' => ['productId', 'integer', 'int32'],
			'Id' => ['id', 'string', null],
			'Name' => ['name', 'string', null],
			'Count' => ['count', 'integer', 'int32'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'price' => ['Price', 'number', 'double'],
			'onlinePrice' => ['OnlinePrice', 'number', 'double'],
			'taxIncluded' => ['TaxIncluded', 'number', 'double'],
			'programId' => ['ProgramId', 'integer', 'int32'],
			'taxRate' => ['TaxRate', 'number', 'double'],
			'productId' => ['ProductId', 'integer', 'int32'],
			'id' => ['Id', 'string', null],
			'name' => ['Name', 'string', null],
			'count' => ['Count', 'integer', 'int32'],
			];
	}
}
